frontend/packages/console-app/locales/ja/volume-snapshot-content.json

```diff
@@ -0,0 +1,14 @@
+{
+  "VolumeSnapshotContent details": "ボリュームスナップショットコンテンツの詳細",
+  "Status": "ステータス",
+  "Size": "サイズ",
+  "VolumeSnapshot": "VolumeSnapshot",
+  "VolumeSnapshotContent": "VolumeSnapshotContent",
```

Comment:
There is no "VolumeSnapshotContent" field on VolumeSnapshotContent detail page, the field should be "VolumeSnapshotClass". Refer to this bug: https://bugzilla.redhat.com/show_bug.cgi?id=1944590

Reply:
It looks like this string is used in volume-snapshot-content-details.tsx and volume-snapshot-details.tsx.

Reply:
"VolumeSnapshotClass" is used in  volume-snapshot-content-details.tsx and and volume-snapshot-details.tsx. Not sure if it matters for translation.

Reply:
I looked again and it looks like VolumeSnapshotContent is being used in volume-snapshot-details.tsx. Doesn't seem to be a bug to me, but let me know if there's something I'm missing.